### PR TITLE
Fix builds for release branch

### PR DESCRIPTION
**Pull request by Hanzei**
Builds are done stable12.

One more question remains: What does https://github.com/nextcloud/twofactor_u2f/blob/master/.travis.yml#L54 do? Do we need this?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,13 +21,15 @@ services:
...
-  - CORE_BRANCH=master
+  - CORE_BRANCH=stable12
   - PHP_COVERAGE=TRUE
   matrix:
   - DB=sqlite
+  - DB=pgsql
 
 branches:
   only:
+  - release
```
Comment (ChristophWurst) [inline .travis.yml]: Why a release branch? I usually just have one main master branch and feature branches for features and bugfixes. Releases are tagged commits on the master branch.

File: .travis.yml [hunk 2]
```diff
@@ -19,13 +21,15 @@ services:
 
 env:
   global:
-  - CORE_BRANCH=master
```
Comment (ChristophWurst) [inline .travis.yml]: we should test against master too

File: .travis.yml [hunk 3]
```diff
@@ -84,12 +88,3 @@ script:
   - if [[ "$PHP_COVERAGE" = "TRUE" ]]; then wget https://scrutinizer-ci.com/ocular.phar; fi
   - if [[ "$PHP_COVERAGE" = "TRUE" ]]; then php ocular.phar code-coverage:upload --format=php-clover clover.xml; fi
 
-matrix:
```
Comment (ChristophWurst) [inline .travis.yml]: why did you remove the matrix? We have at least three builds: the latest release of the Nextcloud server, its current master branch and the js tests.

Comment (ChristophWurst) [inline .travis.yml]: Moreover we use the matrix to test against both php5.6 and php7.0 as well as pgsql, sqlite and mysql.

**Comment by Hanzei** [comment]
@ChristophWurst I thought you proposed to add a release branch. Without a release branch this pr doesn't make sense because its from release_fix-build to release. I will submit a new one.
